### PR TITLE
SAAS-111 Update dbaas-api dependency

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/SAAS-111

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -18,7 +18,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.1
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/hashicorp/go-version v1.2.1 // indirect
-	github.com/percona-platform/dbaas-api v0.0.0-20200804080832-36f9f35f4e21
```
Comment (artemgavrilov) [inline go.mod:21]: TODO: switch to the latest

**Comment by artemgavrilov** [comment]
Updating deps

File: tests/go.mod [hunk 2]
```diff
@@ -5,13 +5,13 @@ go 1.14
 require (
 	github.com/kr/text v0.2.0 // indirect
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
-	github.com/percona-platform/dbaas-api v0.0.0-20200805123334-c3b702bb2287
+	github.com/percona-platform/dbaas-api v0.0.0-20200909123013-067d66ee3244
```
[the remark this answers is not in the document]
Comment (artemgavrilov) [reply tests/go.mod]: Done

File: go.mod (hunk 1 above)
Comment (artemgavrilov) [reply go.mod:21]: Done

**Review by AlekSi — APPROVED** [review]

**Pull request merged**